docs/qa/v034/README.md

```diff
@@ -85,12 +87,18 @@ tiny axis intervals. The cluster shown below looks like a single point in the pl
 
 The plot of latencies can we used as a baseline to compare with other releases.
 
-## Prometheus Metrics on the Chosen Experiment
+The following plot summarizes average latencies versus overall throughputs
```

Comment:
@thanethomson , do you agree to move line 88 below your paragraph (and make it plural). I believe the plot you have introduced should also used in the comparison against the baseline.